code/game/machinery/constructable_frame.dm

```diff
@@ -1074,7 +1075,10 @@ to destroy them and players will be able to make replacements.
 							/obj/item/stack/sheet/glass = 1)
 
 /obj/item/circuitboard/merch
-	name = "Merchandise Computer Circuitboard"
+	board_name = "Merchandise Computer Circuit Board"
```

Comment:
```suggestion
	board_name = "Merchandise Computer"
```
It will automatically add "circuit board" into the name
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/42c3f530-80c3-4056-95bc-0c38a2c917db)

Reply:
I'm contemplating changing the board name to Nanotrasen Merchandise Vendor to match the name of the machine that is made as the vast majority of boards either directly match or have some minor differences between their name and the machine they make.